README-Milestone2.md

```diff
@@ -0,0 +1,72 @@
+# Milestone 2 - Integrate a Web Application
+In this section, we will connect a web application to an existing Uni
+
+## Explore the /client_app directory
+The /client_app directory contains a pre-crated React web application that will help us quickly integrate a user-facing application with a Uni
+
+* __package.json__ - Contains the configuration for the web application, including a series of scripts that can be executed from the command line.
+* __gql__ - A set of GraphQL operations (Queries and Mutations) that can be used to create/read/update/delete/list characters
+
+## Modify the Configuration
+Now it's time to integrate with the Uni from the previous section.
+
+We can get the GraphQL configuration from the Vendia Share UI or with the `share` CLI.
+
+```bash
+
+share get --uni <name_of_your_uni>
+
+```
+
+Modify the `share_env.json` file to reference the correct GraphQL endpoint and API Key.
```

Comment:
modifying share_env.json is not needed. the npm script does this.

Reply:
Didn't realize that.  Thank you!